src/coreclr/debug/createdump/crashinfounix.cpp

```diff
@@ -378,7 +378,7 @@ CrashInfo::VisitProgramHeader(uint64_t loadbias, uint64_t baseAddress, Phdr* phd
 
             uint64_t ehFrameStart;
             uint64_t ehFrameSize;
-            if (PAL_GetUnwindInfoSize(baseAddress, ehFrameHdrStart, ReadMemoryAdapter, &ehFrameStart, &ehFrameSize))
+            if (PAL_GetUnwindInfoSize(baseAddress, ehFrameHdrStart, ReadMemoryAdapter, (ULONG64*)&ehFrameStart, (ULONG64*)&ehFrameSize))
```

Comment:
Change the type of `ehFrameStart` to avoid the casts?

Reply:
It felt ugly as the method has other locals of uint64_t type and the surrounding methods also use these types.

Reply:
I would take this ugliness over casts like this. We had many bugs over the years where the cast like this ended up casting to larger or smaller type, leading to stack smashing bug or uninitialized memory bug.

Reply:
Ok, done